src/test/groovy/com/github/rodionovsasha/commentservice/services/UserServiceTest.groovy

```diff
@@ -0,0 +1,173 @@
+package com.github.rodionovsasha.commentservice.services
+
+import com.github.rodionovsasha.commentservice.BaseTest
+import com.github.rodionovsasha.commentservice.exceptions.InactiveUserException
+import com.github.rodionovsasha.commentservice.exceptions.UserNotFoundException
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.dao.EmptyResultDataAccessException
+
+class UserServiceTest extends BaseTest {
```

Comment:
Let's have methods naming convention like

<method_name> <does|returns|throws> when <condition(s)>

For example,

* "deactivates makes user inactive"
* "activate throws when user is not found by id"
and so on.